distribution/osbuild-dnf-json.socket

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=OSbuild dnf-json socket
```

Comment:
The tiniest nitpick ever: One unit has `OSBuild` in description, the other one has `OSbuild`.